CodeStarterCamp_Week4/main.swift

```diff
@@ -8,5 +8,110 @@
+            if fatigueDegree > oldValue {
+                print("피로도가 \(fatigueDegree - oldValue) 상승합니다.")
+            } else {
+                print("피로도가 \(oldValue - fatigueDegree) 하락합니다.")
+            }
+        }
+    }
+    var coreStrength: Int {
+        
+        didSet(oldValue) {
+            print("코어힘이 \(coreStrength - oldValue) 상승합니다.")
+        }
+    }
+    var bodyWeight: Int = 100 {
+        
+        willSet(afterDiet) {
+            print("\(bodyWeight)kg 에서 \(afterDiet-bodyWeight)kg 감량했습니다.")
+        }
+    }
+    init(upperBodyStrength: Int, lowerBodyStrength: Int, muscularEndurance : Int, fatigueDegree: Int, coreStrength: Int) {
+        self.upperBodyStrength = upperBodyStrength
+        self.lowerBodyStrength = lowerBodyStrength
+        self.muscularEndurance = muscularEndurance
+        self.fatigueDegree = fatigueDegree
+        self.coreStrength = coreStrength
+    }
+    func checkBodyCondition() {
+        print("--------------")
+        print("현재의 컨디션은 다음과 같습니다.")
+        print("상체근력: \(upperBodyStrength)")
+        print("하체근력: \(lowerBodyStrength)")
+        print("근지구력: \(muscularEndurance)")
+        print("피로도: \(fatigueDegree)")
+        print("코어힘: \(coreStrength)")
+        print("--------------")
+        print("최종 몸무게는 \(bodyWeight)kg 입니다.")
+    }
+}
+
+struct Activity {
+    let name : String
+    let action: (BodyCondition) -> Void
+}
+
+let situp: Activity = Activity(name: "윗몸일으키기", action: { bodycondition in
+    print("<<\(situp.name)을(를) 시작합니다>>")
+    bodycondition.upperBodyStrength += Int.random(in: 10...20)
+    bodycondition.fatigueDegree += Int.random(in: 10...20)
+})
+let squirt: Activity = Activity(name: "스쿼트", action: { bodycondition in
+    print("<<\(squirt.name)을(를) 시작합니다>>")
+    bodycondition.lowerBodyStrength += Int.random(in: 20...30)
+    bodycondition.fatigueDegree += Int.random(in: 10...20)
+    bodycondition.bodyWeight -= Int.random(in: 1...10)
+})
+let longDistanceRunning: Activity = Activity(name: "오래달리기", action: { bodycondition in
+    print("<<\(longDistanceRunning.name)을(를) 시작합니다>>")
+    bodycondition.muscularEndurance += Int.random(in: 20...30)
+    bodycondition.upperBodyStrength += Int.random(in: 5...10)
+    bodycondition.lowerBodyStrength += Int.random(in: 5...10)
+    bodycondition.fatigueDegree += Int.random(in: 20...30)
+})
+let dynamicRest: Activity = Activity(name: "동적휴식", action: { bodycondition in
+    print("<<\(dynamicRest.name)을(를) 시작합니다>>")
+    bodycondition.fatigueDegree -= Int.random(in: 5...10)
+})
+let pilates: Activity = Activity(name: "필라테스", action: { bodycondition in
+    print("<<\(pilates.name)을(를) 시작합니다>>")
+    bodycondition.coreStrength += Int.random(in: 5...10)
+    bodycondition.fatigueDegree += Int.random(in: 5...10)
+    bodycondition.bodyWeight -= Int.random(in: 1...10)
+})
+
+var yourBodyCondition: BodyCondition = BodyCondition(upperBodyStrength: 0, lowerBodyStrength: 0, muscularEndurance: 0, fatigueDegree: 0, coreStrength: 0)
+
```

Comment:
```suggestion

var yourBodyCondition: BodyCondition = BodyCondition(
    upperBodyStrength: 0,
    lowerBodyStrength: 0,
    muscularEndurance: 0,
    fatigueDegree: 0,
    coreStrength: 0
)

```

함수 호출시에도 마찬가지로 파라미터가 3개 이상일 경우, 이런식으로 줄바꿈을 활용해주기도 합니다